static/app/views/replays/detail/domMutations/index.tsx

```diff
@@ -14,11 +13,13 @@ import type ReplayReader from 'sentry/utils/replays/replayReader';
 
 type Props = {
   replay: ReplayReader;
+  width: number;
 };
 
-function DomMutations({replay}: Props) {
+function DomMutations({replay, width}: Props) {
   const {isLoading, actions} = useExtractedCrumbHtml({replay});
   const startTimestampMs = replay.getReplay().startedAt.getTime();
+  const isWideView = width > 768;
```

Comment:
good idea to hoist magic constants up to the top of the file and give them a name. And/or make this a method with a name

Reply:
There's at least one other tab that would be interested in that information (IssueList) since I saw a media query in it. I almost was going to just add an id to the tab and try to do it the vanilla javascript way instead of the react way to avoid re renders but I decided against it and went this route and just made sure to debounce the resize observer callback